src/beautifiers/latex-beautify.coffee

```diff
@@ -0,0 +1,72 @@
+"use strict"
+Beautifier = require('./beautifier')
+path = require('path')
+fs = require("fs")
+temp = require("temp").track()
+
+
+module.exports = class LatexBeautify extends Beautifier
+  name: "Latex Beautify"
```

Comment:
Should this not be `LaTeX Indent` instead of `beautify` since that is what it is called in https://github.com/cmhughes/latexindent.pl ?